Packs/McAfeeNSM/Integrations/McAfeeNSMv2/McAfeeNSMv2.py

```diff
@@ -0,0 +1,1946 @@
+    """
+    domain_id = int(args.get('domain_id', None))
+    limit = arg_to_number(args.get('limit', 50)) or 50
+    page = arg_to_number(args.get('page'))
+    page_size = arg_to_number(args.get('page_size'))
+    if (page and not page_size) or (not page and page_size):
+        raise Exception('Please provide both page and page_size arguments.')
+    if args.get('limit') and page:
+        raise Exception('Please provide either limit argument or page and page_size arguments.')
+
+    response = client.list_domain_firewall_policy_request(domain_id)
+    result = sorted(response.get('FirewallPoliciesForDomainResponseList', []), key=lambda k: k['policyId'], reverse=True)
+    result = pagination(result, limit, page, page_size)
+    human_readable = []
+    for value in result:
+        d = {'policyId': value.get('policyId'),
+             'policyName': value.get('policyName'),
+             'domainId': value.get('domainId'),
+             'visibleToChild': value.get('visibleToChild'),
+             'description': value.get('description'),
+             'isEditable': value.get('isEditable'),
+             'policyType': value.get('policyType'),
+             'policyVersion': value.get('policyVersion'),
+             'lastModUser': value.get('lastModUser')}
+        human_readable.append(d)
+
+    hr_title = 'Firewall Policies List'
+    headers = ['policyId', 'policyName', 'domainId', 'visibleToChild', 'description', 'isEditable', 'policyType',
+               'policyVersion', 'lastModUser']
+    readable_output = tableToMarkdown(
+        name=hr_title,
+        t=human_readable,
+        removeNull=True,
+        headers=headers
+    )
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='NSM.Policy',
+        outputs_key_field='policyId',
+        outputs=result,
+        raw_response=result
+    )
+
+
+def get_firewall_policy_command(client: Client, args: Dict) -> CommandResults:
+    """ Gets the Firewall Policy details.
+    Args:
+        client: client - A McAfeeNSM client.
+        args: Dict - The function arguments.
+    Returns:
+        A CommandResult object with the Firewall Policy details.
+    """
+    policy_id = int(args.get('policy_id', ''))
+    include_rule_objects = argToBoolean(args.get('include_rule_objects', False))
+    response = client.get_firewall_policy_request(policy_id)
+    if not include_rule_objects:
+        member_rule_list = response.get('MemberDetails', {}).get('MemberRuleList', [Dict])
+        for member in member_rule_list:
+            del member['SourceAddressObjectList']
+            del member['DestinationAddressObjectList']
+            del member['SourceUserObjectList']
+            del member['ServiceObjectList']
+            del member['ApplicationObjectList']
+            del member['TimeObjectList']
+    human_readable = {'FirewallPolicyId': response.get('FirewallPolicyId'),
```

Comment:
```suggestion
    contents = {'FirewallPolicyId': response.get('FirewallPolicyId'),
```
Should be a better name